CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ This component adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.h
 
 ### Added
 
+- Support `OTEL_LOG_LEVEL` to configure SDK logging level.
```

Comment:
Is `OTEL_DOTNET_AUTO_DEBUG` no longer supported? If so then we should document it and also update the docs.

Reply:
updated in https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/2288/commits/234818f2b9e74db166159f24540019dbb26cd8ec